IaC/rel.cd/JenkinsStack.yml

```
@@ -712,28 +706,33 @@ Resources:
                   wget -O /mnt/$JENKINS_HOST/nginx-google-oauth.lua \
                     https://raw.githubusercontent.com/cloudflare/nginx-google-oauth/master/access.lua
 
-                  chkconfig openresty on
-                  service openresty start
+                  systemctl enable --now openresty
               }
 
               setup_letsencrypt() {
                   install -d /usr/share/nginx/html
-                  certbot --debug --non-interactive certonly --agree-tos --register-unsafely-without-email --webroot -w /usr/share/nginx/html --keep -d $JENKINS_HOST
+                  if [[ -d /mnt/ssl_backup ]]; then
+                    rm -rf /etc/letsencrypt
+                    rsync -aHSv /mnt/ssl_backup/ /etc/letsencrypt/
+                    certbot renew
+                  else
+                    certbot --debug --non-interactive certonly --agree-tos --register-unsafely-without-email --webroot -w /usr/share/nginx/html --keep -d $JENKINS_HOST
+                  fi
                   ln -f -s /etc/letsencrypt/live/$JENKINS_HOST/fullchain.pem /etc/nginx/ssl/certificate.crt
                   ln -f -s /etc/letsencrypt/live/$JENKINS_HOST/privkey.pem   /etc/nginx/ssl/certificate.key
-                  printf '#!/bin/sh\ncertbot renew\nservice openresty reload\n' > /etc/cron.daily/certbot
+                  printf '#!/bin/sh\ncertbot renew\nsystemctl reload openresty\nrm -rf  /mnt/ssl_backup\nrsync -aHSv /etc/letsencrypt/ /mnt/ssl_backup/\n' > /etc/cron.daily/certbot
```

I believe `systemctl restart openresty` should be used here
```suggestion
                  printf '#!/bin/sh\ncertbot renew\nsystemctl restart openresty\nrsync -aHSv --delete /etc/letsencrypt/ /mnt/ssl_backup/\n' > /etc/cron.daily/certbot
```

---

Fixed